.drone.star

```diff
@@ -177,7 +176,6 @@ basicTestSuites = [
     "webUIFilesDetails",
     "webUILogin",
     "webUIMoveFilesFolders",
-    "webUIPreview",
```

Comment:
suite is still there. please remove it
https://github.com/owncloud/web/blob/0b4f5baa9853a64b2811ff3fd9302081de08bdb9/tests/acceptance/features/webUIPreview/mediaPreview.feature